Doc/library/decimal.rst

```diff
@@ -88,8 +88,8 @@ context for arithmetic, and signals.
 A decimal number is immutable.  It has a sign, coefficient digits, and an
 exponent.  To preserve significance, the coefficient digits do not truncate
 trailing zeros.  Decimals also include special values such as
-:const:`Infinity`, :const:`-Infinity`, and :const:`NaN`.  The standard also
-differentiates :const:`-0` from :const:`+0`.
+``Infinity``, ``-Infinity``, and ``NaN``.  The standard also
```

Comment:
Also, if we're going to use a literal here, maybe worth using the actual literal that is printed (``inf`` and ``-inf``) in the places ``Infinity`` is currently used?

Reply:
Is it referring to things like `Decimal('Infinity')`, `Decimal('-Infinity')` and `Decimal('NaN')`?

Reply:
> Is it referring to things like Decimal('Infinity'), Decimal('-Infinity') and Decimal('NaN')?

Yes, and those are printed as shown:

```python
>>> print(Decimal('inf'))
Infinity
>>> print(Decimal('-inf'))
-Infinity
>>> print(Decimal('nan'))
NaN
```

I don't think we should change from `Infinity` to `inf` here.

Reply:
Ah, gotcha—didn't realize that `Decimal` accepted literals in the form `'Infinity'`/`'-Infinity'` rather than the `inf` and `-inf` spelling printed by `float`, so the current form makes perfect sense. Thanks @mdickinson !